src/components/subscription-choices/subscription-choices.scss

```diff
@@ -0,0 +1,72 @@
+@use '../../compiled/tokens/scss/breakpoint';
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/size';
+@use '../../mixins/a11y';
+@use '../../mixins/theme';
+
+///
+/// Theme properties
+///
+
+// Light theme
+@include theme.props {
+  --theme-color-background-subscription-form: #{color.$base-white};
+}
+
+// Dark theme
+@include theme.props(dark) {
+  --theme-color-background-subscription-form: #{color.$brand-primary};
+}
+
+// Alternate theme
+@include theme.props(alternate) {
+  --theme-color-background-subscription-form: #{color.$base-gray-lighter};
+}
+
+///
+/// Subscription Choices
+///
+
+.c-subscription-choices {
+  position: relative;
+}
+
+.c-subscription-choices__controls-wrapper {
+  display: grid;
```

Comment:
I wonder if I should use `flex` here instead. It seems a bit much to use `grid` for managing two buttons? 🤔

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1679/commits/975e09a519bea456fba87a65488739160e16e779